docs/excel/build-your-first-excel-add-in.md

```diff
@@ -10,7 +10,9 @@ You’ll start by creating a web app using HTML and JQuery. Then you'll create a
 
 ### Code it
 
-1- Create a folder on your local drive named QuarterlySalesReport (for example, C:\\QuarterlySalesReport). Save all files created in the steps that follow into this folder.
+#### Create the web app
```

Comment:
And actually, it looks like we don't currently have H2s in this article. We should elevate to H2s to help with the navigation.